apps-rendering/src/components/editions/headerImageCaption.tsx

```diff
@@ -85,6 +85,8 @@ const toReact = (node: Node, key: number): ReactNode => {
 	switch (node.nodeName) {
 		case 'EM':
 			return <em>{node.textContent}</em>;
+		case 'A':
+			return <span>{node.textContent ?? ''}</span>;
```

Comment:
What do we get from the empty string here? If its needed here, should we add it to the EM switch as well?

Reply:
Not much tbh, was just copying from another implementation but it works fine without.